shuup/simple_supplier/admin_module/views.py

```diff
@@ -138,6 +139,10 @@ def _process_stock_adjustment(form, request, supplier_id, product_id):
     stock_adjustment = supplier.adjust_stock(
         product_id, delta=data.get("delta"), purchase_price=data.get("purchase_price"), created_by=request.user
     )
+    if not stock_adjustment:
+        raise MissingSupplierModuleException(
+            _("Please be in contact with the marketplace owner to get permission to update your product stock")
```

Comment:
makes no sense to raise this here. this view shouldn't even be used if the product doesn't support the simple_supplier module